lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9092,7 +9093,37 @@ class SwiftifyInfoPrinter {
     out << "]";
   }
 
+  void printAvailability() {
+    printSeparator();
+    out << "availability: [";
+    printAvailabilityOfType("Span");
+    printSeparator();
+    printAvailabilityOfType("MutableSpan");
+    out << "]";
+  }
+
 private:
+  ValueDecl *getDecl(StringRef DeclName) {
+    SmallVector<ValueDecl *, 1> decls;
+    SwiftContext.lookupInSwiftModule(DeclName, decls);
+    assert(decls.size() == 1);
+    if (decls.size() != 1) return nullptr;
+    return decls[0];
+  }
+
+  void printAvailabilityOfType(StringRef Name) {
+    ValueDecl *D = getDecl(Name);
+    out << "\"" << Name << "\":" << "\"";
+    llvm::SaveAndRestore<bool> hasAvailbilitySeparatorRestore(firstParam, true);
+    for (auto attr : D->getSemanticAvailableAttrs(/*includingInactive=*/true)) {
```

Comment:
We don't need the inactive availability attributes, do we?

Reply:
Oh right, I forgot to test whether that made a difference

Reply:
I tried this out now, and while we don't _necessarily_ need to include inactive attributes, I think it kind of makes sense to include them. If we don't, we'll get `@available(macOS 10.14.4, *)` when compiling for macOS, other stuff for other Darwin platforms, and nothing for Linux and Windows. If we include inactive attributes we'll always get `@available(visionOS 1.1, tvOS 12.2, watchOS 5.2, iOS 12.2, macOS 10.14.4, *)`. This makes the output more consistent, making it significantly easier to test, and less confusing for someone reading the macro expansion.

Reply:
Alright, that makes sense.